docs/InstallGuide.md

```diff
@@ -131,294 +158,206 @@ To automatically launch the mysql server, execute the following command.
+* mode
 
-    # chkconfig --list mysqld
-    mysqld          0:off   1:off   2:off   3:off   4:off   5:off   6:off
-    # chkconfig mysqld on
-    mysqld          0:off   1:off   2:on    3:on    4:on    5:on    6:off
-    # service mysqld start
-
-
-Start OpenVNet
----------------
-    # reboot
-    #
-    # initctl start vnet-vnmgr
-    # initctl start vnet-webapi
-    # initctl start vnet-vna
-    # initctl list | grep vnet # <- This line confirms that all three processes have started successfully.
-    vnet-vna start/running, process 2505
-    vnet-vnmgr start/running, process 2286
-    vnet-webapi start/running, process 2296
-
-
-Download test environment
---------------------------
-
-Download the test environment archive and unpack it in the /root directory. The size of this archive is about 1.3 GB.
-
-     # curl -o /root/vnet-test-kvm.tgz -R http://dlc.openvnet.axsh.jp/tests/vnet-test-kvm.tgz
-     # tar xvfz ./vnet-test-kvm.tgz
+The mode of the interface. `vif` must be specified if a virtual machine is supposed to have this.
 
-Create Database
-----------------
+* owner-datapath-uuid
 
-Create the database and enter the sample data.
+The UUID of the datapath to which this interface will be connected.
 
-    # vi ~/.bash_profile
-    PATH=$PATH:$HOME/bin -> PATH=$PATH:$HOME/bin:/opt/axsh/openvnet/ruby/bin
-    # source ~/.bash_profile
-    # cd /root/vnet-test-kvm
-    # ./db.sh
+* mac-address
 
+The MAC address of an interface of a virtual machine.
 
-Reboot vna
------------
+* network-uuid
 
-    # initctl restart vnet-vna
-    # initctl list | grep vnet # <- This line confirms that all three process have started successfully.
-    vnet-vna start/running, process 2699
-    vnet-vnmgr start/running, process 2286
-    vnet-webapi start/running, process 2296
+The UUID of the virtual network to participate
 
+* ipv4-address
 
-Check setup for OpenFlow
--------------------------
+The IPv4 address which will be assigned to the network interface.
 
-### openvswitch process
+* port-name
 
-    # /etc/init.d/openvswitch status
-    ovsdb-server is running with pid 868
-    ovs-vswitchd is running with pid 877
+OpenVNet needs to know which port of the Open vSwitch is associated to the database. You can see what is attached by `ovs-vsctl show`.
 
-### bridge
+```
+fbe23184-7f14-46cb-857b-3abf6153a6d6
+    Bridge "br0"
+        Controller "tcp:127.0.0.1:6633"
+            is_connected: true
+        fail_mode: standalone
+        Port "br0"
+            Interface "br0"
+                type: internal
+```
 
-    # ovs-vsctl show
-    70cf0a71-9413-4889-867c-d76d8e2b7ec4
-        Bridge "br0"
-            Controller "tcp:127.0.0.1:6633"
-            fail_mode: standalone
-            Port "eth0"
-                Interface "eth0"
-            Port "br0"
-                Interface "br0"
-                    type: internal
-        ovs_version: "1.10.0"
+There are 2 ports named as inst1 and inst2 on the example output above. They are must be the same
```

Comment:
I think this is confusing because the ports inst1 and inst2 don't exist yet. Perhaps it's a better idea to start the VMs before installing OpenVNet.